src/payment/checkout/payment-form/PaymentForm.jsx

```diff
@@ -198,13 +203,30 @@ export class PaymentFormComponent extends React.Component {
       isQuantityUpdating,
     } = this.props;
 
+    // Stripe element config
+    // TODO: Move these to a better home
+    const appearance = {
+      theme: 'stripe',
+    };
+    const options = {
+      clientSecret: this.props.captureKeyId,
+      appearance,
+    };
+    const tempFlag = false;
```

Comment:
I'm retrieving the flag in the parent component `<Checkout />`, I can add props to pass this in to `<PaymentForm />`, calling it `stripeEnabled` (accessible through `this.props`)

Reply:
PR https://github.com/openedx/frontend-app-payment/pull/638